### PR TITLE
[Document][202106] Supply the missing ingress/egress port profile list in document

### PR DESCRIPTION
**Pull request by stephenxs**
**What I did**

Supply the missing ingress/egress port profile list tables in the document

Signed-off-by: Stephen Sun stephens@nvidia.com
**Why I did it**

**How I verified it**

**Details if related**

### CODE REVIEW

**Review by keboliu — APPROVED** [review]

File: doc/Configuration.md [hunk 1]
```diff
@@ -504,6 +506,43 @@ This kind of profiles will be handled by buffer manager and won't be applied to
  
 ```
 
+### Buffer port ingress profile list
+
+```
+{
+"BUFFER_PORT_[BUFFER_PROFILE|INGRESS_PROFILE_LIST": {
```
Comment (neethajohn) [inline doc/Configuration.md:513]: Table name needs to be fixed

Comment (stephenxs) [reply doc/Configuration.md]: Fixed.

**Review by neethajohn — APPROVED** [review]

**Comment by stephenxs** [comment]
Currently, the PR depends on #2118 for the vstest failure.

**Pull request closed without merging**